code/__DEFINES/misc.dm

```diff
@@ -471,3 +472,10 @@ GLOBAL_LIST_EMPTY(monkey_icon_cache)
 // Pirates threat
 #define PIRATE_RESPONSE_NO_PAY "pirate_answer_no_pay"
 #define PIRATE_RESPONSE_PAY "pirate_answer_pay"
+
+//NSV13 - AI Custom Holographic Form - Start
+#define CUSTOM_HOLOFORM_DELAY 10 SECONDS //prevents spamming to make lag. it's pretty expensive to do this.
+
+#define HOLOFORM_FILTER_AI "FILTER_AI"
+#define HOLOFORM_FILTER_STATIC "FILTER_STATIC"
+//NSV13 - AI Custom Holographic Form - Stop
```

Comment:
Defines can be moved to the NSV defines as well

Reply:
Can't move those as it produces a compile error if moved to an NSV file

Reply:
Never mind